central/scannerdefinitions/handler/handler.go

```diff
@@ -517,8 +418,8 @@ func (h *httpHandler) openMostRecentV4OnlineFile(_ context.Context, t updaterTyp
 	}
 	log.Debugf("Compressed data file is available: %s", openedFile.Name())
 	switch t {
-	case mappingUpdaterType:
-		targetFile, cleanUp, err := openFromArchive(openedFile.Name(), opts.mappingFile)
+	case mappingUpdaterType, v2UpdaterType:
```

Comment:
Doesn't look like `v2UpdaterType` belongs here.  Tested this out with the image from and V2 is broken:


```
{"Event":"Updater failed","Level":"error","Location":"updater.go:86","Time":"2024-07-23 20:11:31.329024","error":"fetching update from URL: received response code 500 Internal Server Error, but expected 2xx; error message: {\"code\":13,\"message\":\"could not read vulnerability definition fb5a20c3-7c89-42da-a7ef-4678261a096f: extracting: open : invalid argument\"}"}
```